server/controllers/post.js

```diff
@@ -0,0 +1,77 @@
+const { GeneralError, NotFound } = require('../utils/errors');
+const { validationResult } = require('express-validator');
+
+const Profile = require('../models/profile');
+const Post = require('../models/post');
+const Forum = require('../models/Forum');
+
+// exports.creatPost = async (req, res, next) => {
```

Comment:
Can this commented code be removed?

Reply:
I thought we might need a route just to create posts in the future but there is a similar route, so, it can be removed.